### PR TITLE
pass className, style to div wrapper

### PR DESCRIPTION
**Pull request by geekingreen**
It might be useful to alter the styles of the div wrapper.

### CODE REVIEW

**Comment by aaronjensen** [comment]
Thanks! I think actually we're planning on moving towards not rendering that div at all. There's not much reason for it now. Would you like to try out just rendering the child directly?

**Comment by geekingreen** [comment]
Well as it stands one could potentially have multiple children (in my use case I do) in the `MediaQueryable` tag. As far as I understand it React doesn't support rendering multiple children without having a parent tag as shown in this issue https://github.com/facebook/react/issues/2127 .

**Comment by aaronjensen** [comment]
ah, I forgot it had that capability. 

To be honest, we'd like to move this to be a higher order component similar to redux's `connect` rather than this model (which is a little behind the times).

I'm good with taking this in until the day that we can change this to a HOC, but can you add `propTypes` please? Thanks!

File: react_media_queryable.js [hunk 1]
```diff
@@ -62,8 +62,10 @@ return /******/ (function(modules) { // webpackBootstrap
 	  displayName: 'MediaQueryable',
 	  mediaListener: null,
 	  propTypes: {
+	    className: React.PropTypes.string,
 	    defaultMediaQuery: React.PropTypes.string.isRequired,
 	    mediaQueries: React.PropTypes.object.isRequired,
+	    style: React.PropTyps.object
```
Comment (aaronjensen) [inline react_media_queryable.js]: typo here

File: src/media_queryable.js [hunk 2]
```diff
@@ -6,8 +6,10 @@ module.exports = React.createClass({
   displayName: 'MediaQueryable',
   mediaListener: null,
   propTypes: {
+    className: React.PropTypes.string,
     defaultMediaQuery: React.PropTypes.string.isRequired,
     mediaQueries: React.PropTypes.object.isRequired,
+    style: React.PropTypes.object
```
Comment (aaronjensen) [inline src/media_queryable.js]: Hate to nitpick on a project w/o eslint, but could you add the trailing comma to match the existing style please? Thanks!

**Comment by geekingreen** [comment]
Not a problem

**Comment by aaronjensen** [comment]
Thanks!

**Pull request merged**